Library/test-helpers/lib.sh

```diff
@@ -1906,62 +1906,21 @@ Returns 0.
 
 =cut
 
-limeconPrepareAgentImage() {
+limeconPrepareImage() {
 
     if [ "$1" == "-f" -o "$1" == "--file" ]; then
         local DOCKER_FILE=$2
-        shift 2
-    else
-        echo "Using default docker file: ${limeLibraryDir}/Dockerfile.agent"
-        DOCKER_FILE="${limeLibraryDir}/Dockerfile.agent"
     fi
 
-    local TAG=$1
-
-    cp /var/lib/keylime/cv_ca/cacert.crt .    
-    ls /root/.ssh/id_*.pub || ssh-keygen -t rsa -N "" -f /root/.ssh/id_rsa
-    cp /root/.ssh/id_*.pub .
-    cp ${limeLibraryDir}/lime_con_start.sh .
-    podman build -t=$TAG --file $DOCKER_FILE .
-}
-
-true <<'=cut'
-=pod
-
-=head2 limeconPrepareRegistrarImage
-
-Prepare podman image with keylime registrar. Install keylime registrar.
-
-    limeconPrepareAgentImage [-f|--file DOCKER_FILE] TAG
-
-=over
-
-=item -f, --file DOCKER_FILE
-
-Parameter specify use of non-default docker file.
-
-=item TAG
-
-Name of image tag.
-
-=back
-
-Returns 0.
-
-=cut
-
-limeconPrepareRegistrarImage() {
-
-    if [ "$1" == "-f" -o "$1" == "--file" ]; then
-        local DOCKER_FILE=$2
-        shift 2
-    else
-        echo "Using default docker file: ${limeLibraryDir}/Dockerfile.registrar"
-        DOCKER_FILE="${limeLibraryDir}/Dockerfile.registrar"
+    if [ "$3" == "-t" -o "$3" == "--tag" ]; then
+        local TAG=$4
     fi
 
-    local TAG=$1
+    if [[ "$DOCKER_FILE" =~ .*"Dockerfile.agent".* ]]; then
```

Comment:
I do not like this hardcoded match.. I was thinking, we could add optiong like `--cacert FILE` which would ensure that a cert from the given path is copied into the keylime container.